src/StdAfterTest.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+// Inheritted by test contracts where afterTest() is needed
+abstract contract StdAfterTest {
+    // This modifier needs to applied to the test after which we want to run the afterTest() function
+    modifier runAftetTest() {
+        _;
+        afterTest();
+    }
+
+    // This is needs to be overriden by the Test contract that inherits this
+    function afterTest() public virtual;
```

Comment:
Can we add some comments to document how this works exactly and what it's intended for? Does it run only after successful tests, or also failed ones? If both, I can see a use case like cleaning up tmp files. What's the use case for only running after successful test cases?

Reply:
`afterTest()` won't run after tests that revert.

Reply:
Makes sense.
Closing this and adding an example impl to the linked issue.